hazelcast/src/main/java/com/hazelcast/monitor/NearCacheStats.java

```diff
@@ -59,4 +59,11 @@
      * @return hit/miss ratio of the locally owned entries.
      */
     double getRatio();
+
+    /**
+     * Returns the number of evictions of locally owned entries.
```

Comment:
`locally owned entries` seems a blurry explanation. Because, in most cases, nearCache is used for caching entries of remote nodes. An alternative doc can be like this: 

`Returns the number of evictions from this near cache`<-- _but this also doesn't address a question: Are expired entries also included in eviction-count?_
_If yes, why don't we provide a separate `getExpirations` method? Because expiration is not eviction in hazelcast parlance._